### PR TITLE
Add support to unmarshal json in cli option struct

### PR DESCRIPTION
**Pull request by abdullah1308**
**Description**

This PR fixes #

**Notes for Reviewers**

Dependabot updates also addressed

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by MUzairS15 — CHANGES_REQUESTED** [review]
@

File: pkg/proto/utils.go [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+		Nanos   int32 `json:"nanos"`
+	}
+
+	var set map[string]interface{}
+
+	if err := json.Unmarshal(data, &set); err != nil {
+		return err
+	}
+
+	o := &struct {
```
[the remark this answers is not in the document]
Comment (abdullah1308) [reply pkg/proto/utils.go:25]: We've had this conversation on slack before, we cannot unmarshall json into the struct `CommandLineOptions` as the fields are of _custom types_ and not _Golang primitive types_.

Example error when trying to unmarshal the following json
```
{
    "requests_per_second": 3
}
```
```
ERRO[2023-07-19T23:43:26+05:30] json: cannot unmarshal number into Go struct field CommandLineOptions.requests_per_second of type wrapperspb.UInt32Value  app=meshery code=2208 probable-cause="Load test endpoint could be not reachable" severity=2 short-description="load test error: .unable to perform" suggested-remediation="Make sure load test endpoint is reachable"
```

The struct expects a value of type `wrapperspb.UInt32Value` for the field `RequestsPerSecond`.

[the remark this answers is not in the document]
Comment (abdullah1308) [reply pkg/proto/utils.go:25]: The structs aren't exactly the same. I've redefined the struct fields as primitives -

image

Then I assign value to the `CommandLineOptions` struct with the proper type -

image

[the remark this answers is not in the document]
Comment (MUzairS15) [reply pkg/proto/utils.go:25]: My bad, It didn’t came to my mind unless I saw that struct carefully that it has non golang primitives.

**Review by MUzairS15 — APPROVED** [review]

**Comment by MUzairS15** [comment]
@abdullah1308 some lint checks to address?

**Pull request merged**